fireant/widgets/reacttable.py

```diff
@@ -543,22 +562,29 @@ def transform(
         }
         # has at least 1 dim and all are pivoted
         all_dimensions_pivoted = len(dimensions) and (len(dimensions) == len(self.pivot))
-
         metric_aliases = list(metric_map.keys())
-        dimension_aliases = [
-            alias_selector(dimension.alias) for dimension in self.pivot
+
+        hide_dimension_aliases = set([
```

Comment:
A set comprehension like `hide_dimension_aliases = {alias_selector(dimension.alias) for dimension in self.hide}` would be more efficient as it wouldn't need to create the list (although I doubt whether that would have a real world performance impact given the usage of transform())

Reply:
Changed to use a set comprehension.